.ddev/config.yaml

```diff
@@ -17,6 +17,19 @@ web_environment:
   - LAGOON_PROJECT=contribkanban
   - LAGOON_ENVIRONMENT=master
 nodejs_version: "16"
+hooks:
+  post-start:
+    - exec: yarn install
+web_extra_daemons:
+  - name: http
+    command: "yarn frontend-start"
+    directory: /var/www/html
```

Comment:
I'd had issues with this, where it runs before dependencies have installed. I'm pretty sure this starts up before `post-start` hook executes.